backend/prisma/schema.prisma

```diff
@@ -0,0 +1,58 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model questionInfoTable {
+  id Int @id @default(autoincrement())
+  pid String?
```

Comment:
Why do we need  `pid` if we have `id`? I'm not sure about the meaning for those two attributes.